packages/shared/src/components/modals/RanksModal/RanksTags.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { ReactElement } from 'react';
+import { ModalSubTitle } from '../common';
+import { RanksTagsProps, RanksTagsSection, RanksTagsList } from './common';
+import RankTagItem from './RankTagItem';
+
+const RanksTags = ({
+  tags,
+  isColorPrimary,
+  limit = 6,
+}: RanksTagsProps): ReactElement => {
+  return (
+    <RanksTagsSection>
+      <ModalSubTitle>Reading status per tag</ModalSubTitle>
+      <RanksTagsList style={{ gridTemplateRows: '1fr 1fr 1fr' }}>
```

Comment:
Still not really sure why we can't just use tailwind classes for this?
I'm pretty sure that rows does this exact same thing?

Reply:
It actually has a different value - https://tailwindcss.com/docs/grid-template-rows

Tried arbitrary value but no luck: `grid-rows-[1fr 1fr 1fr]`.

Reply:
For future reference `repeat(3, minmax(0, 1fr));` is the same as writing `1fr 1fr 1fr`.
The main issue here is this is a new class and the next cache doesn't take these into account.

I'll raise a separate issue for this.
Fixed it now.